build.gradle

```diff
@@ -89,6 +90,12 @@ dependencies {
 }
 
 
+kotlinter {
+  ignoreFailures = false
```

Comment:
The default is true? Very weird.

Reply:
I think the default is false. I tested it wrong and it didn't fail, this was leftover.

Reply:
https://github.com/jeremymailen/kotlinter-gradle#configuration

Yeah it's default is false 🤷‍♂️